src/Aesthetic.ts

```diff
@@ -406,7 +406,7 @@ abstract class Aesthetic<ChannelType extends Channel> {
     return this.default_constant;
   }
 
-  get use_map_on_regl() {
+  get use_map_on_regl() : 1 | 0 {
```

Comment:
If the only possible values are 1 or 0, shouldn't a boolean be used here?

Reply:
This value is being passed straight to the shader as a uniform where it needs to be  cast to a float, so I'm just keeping it in the same type. It's like lower down the jitter format is passed as an integer as well. For clarity might make sense to have a `type WebGLBoolean = 0 | 1` somewhere.

```ts
  public jitter_int_formatted : 0 | 1 | 2 | 3 | 4 | 5 = 0;
```

Reply:
I think that type makes things clearer and is a good idea. Maybe we should add a similar nominal type for jitter too?